src/main/java/frc/robot/subsystems/Hatch.java

```diff
@@ -26,66 +26,83 @@
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
 
-  // initiating three pnumatic solenoid that pushes the hatch out
-  // using three souble solenoids for now
   private DoubleSolenoid m_beak;
   private Solenoid m_launcher;
 
   /*
    * TODO: confirm with the design team about the type and number of solenoids
    * being used
+   * TODO: Verify the CAN id for the PCM and what voltage level is being used
    */
 
   public Hatch() {
-    this.m_launcher = new Solenoid(RobotMap.m_pcm24v, RobotMap.m_launcherExtend);
+    this.m_launcher = new Solenoid(RobotMap.m_pcm12v, RobotMap.m_launcherExtend);
     this.m_beak = new DoubleSolenoid(RobotMap.m_pcm12v, RobotMap.m_beakExtend, RobotMap.m_beakRetract);
   }
 
-  // simple solenoid operation: push out and turn off immediately
-  public void pneumaticsAutoRetract(DoubleSolenoid solenoid) {
-    solenoid.set(DoubleSolenoid.Value.kForward);
-    try {
-      TimeUnit.SECONDS.sleep(1);
-    } catch (InterruptedException e) {
-      e.printStackTrace();
-    }
-    solenoid.set(DoubleSolenoid.Value.kReverse);
+  @Override
+  public void initDefaultCommand() {
+    // Beak need to be in retracted position first before game.
+    setDefaultCommand(new ExtendBeak());
   }
 
-  public void pneumaticsAutoRetract(Solenoid solenoid){
+  /* Single Solenoid Helper functions */
```

Comment:
NOTE: Tried to reorganize the rest of the other functions into groups for better readability.